src/jtag3.c

```diff
@@ -1535,12 +1549,12 @@ static int jtag3_parseextparms(const PROGRAMMER *pgm, const LISTID extparms) {
     // Bit 2 EOF: Agressive power-down, sleep after 5 seconds if no USB enumeration when set to 0
     // Bit 1 LOWP: forces running at 1 MHz when bit set to 0
     // Bit 0 FUSE: Fuses are safe-masked when bit sent to 1 Fuses are unprotected when set to 0
-    else if (str_starts(extended_param, "suffer") ) {
+    if(str_starts(extended_param, "suffer")) {
```

Comment:
`suffer`, `vtarg_switch` and `vtarg` extended options seem to be ignored when the programmer does not have the corresponding feature:

For example,

``` C
   if(str_starts(extended_param, "suffer")) {
     if(pgm->extra_features & HAS_SUFFER) {
       // Process option
     }
   }
```

If you don't want that, there are two possible ways out:
``` C
   if(str_starts(extended_param, "suffer") && pgm->extra_features & HAS_SUFFER) {
     // Process option
   }
```
In this case we get the default error message `pmsg_error("invalid extended parameter %s\n", extended_param);`

Or, alternatively, with a bespoke error message
``` C
   if(str_starts(extended_param, "suffer")) {
     if(pgm->extra_features & HAS_SUFFER) {
       // Process option (ending in continue/break)
     }
     pmsg_error("extended parameter %s used despite programmer not having this feature\n", extended_param);`
     rv = -1;
     break;
   }
```

Reply:
OK, I didn't think this through well enough. The code is OK and gives an error message. My first suggested "solution" is equivalent to existing code.